packages/p2p/src/pages/buy-sell/currency-dropdown.scss

```diff
@@ -42,4 +42,8 @@
             }
         }
     }
+
+    .dc-dropdown__label {
+        transform: translate(0, -0.7rem) scale(0.75);
+    }
```

Comment:
is this needed?

Reply:
yeah, or else the label isnt aligned properly